schemas/UserProperties.json

```diff
@@ -59,6 +59,14 @@
           {"const": "Group", "description": "Modern layout."},
           {"const": "Compact", "description": "Modern layout with compact option enabled."}
         ]
+      },
+      "cryptoModule": {
+        "description": "Which crypto module is the client currently using.",
```

Comment:
I'm not sure this fits well into UserProperties as it'll vary per-client. Clients will clobber each other's cryptoModule field

Reply:
It seems we have the same issue with `WebLayout`, and maybe others too. Where else would we have device-dependent properties?

Alternatively we will primarily be using this field with E2EE Error type, so it could be exposed as a field only on that one event, wdyt @BillCarsonFr ?

Reply:
Web's layout is an account-wide setting, it syncs between your devices

Reply:
Oh in that case the description of that field may be inacurate. I'm also not sure if `allChatsActiveFilter` syncs or not

Reply:
Ah sorry my bad, layout on web used to be account specific, now it is device specific. So yes has the same issue, but clobbering between devices of different kinds is worse IMO. Maybe adding one per platform as a user is likely to only have one Android device etc